src/backgroundblurprocessor/BackgroundBlurProcessorProvided.ts

```diff
@@ -47,6 +47,20 @@ export default class BackgroundBlurProcessorProvided
     this.logger.info('BackgroundBlur processor successfully created');
     this.logger.info(`BackgroundBlur spec: ${this.stringify(this.spec)}`);
     this.logger.info(`BackgroundBlur options: ${this.stringify(options)}`);
+
+    screen.orientation.addEventListener('change', _event => {
+      if (
```

Comment:
Does this event listener need to be removed?

Reply:
Yes, I will do that